utilities/dockerfiles/wildfly/Dockerfile

```
@@ -0,0 +1,46 @@
+ENV WILDFLY_VERSION wildfly-latest
+#ENV WILDFLY_SHA1 eaef7a87062837c215e54511c4ada8951f0bd8d5
+ENV JBOSS_HOME /opt/jboss/wildfly
+
+USER root
+RUN rm /etc/localtime \
+   && ln -s /usr/share/zoneinfo/Asia/Kolkata /etc/localtime
+RUN cd $HOME
+RUN yum install -y epel-release
+RUN yum install -y net-tools
+RUN yum install -y curl cabextract xorg-x11-font-utils fontconfig
+RUN curl -O http://li.nux.ro/download/nux/dextop/el7/x86_64/nux-dextop-release-0-5.el7.nux.noarch.rpm
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)